easybuild/easyconfigs/v/VTK/VTK-6.3.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'VTK'
+version = '6.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.vtk.org'
+description = """The Visualization Toolkit (VTK) is an open-source, freely available software system for
+ 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several
+ interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization
+ algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques
+ such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://www.vtk.org/files/release/%(version_major_minor)s']
+sources = [
+    SOURCE_TAR_GZ,
+    '%(name)sData-%(version)s.tar.gz',
+]
+
+builddependencies = [('CMake', '3.5.2')]
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('libGLU', '9.0.0', '-Mesa-11.2.1'),
+    ('libXt', '1.1.5'),
+]
+
+configopts = "-DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.%s " % SHLIB_EXT
+configopts += "-DOPENGL_gl_LIBRARY=$EBROOTMESA/lib/libGL.%s " % SHLIB_EXT
+configopts += "-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include -DVTK_WRAP_PYTHON=ON "
+configopts += "-DPYTHON_LIBRARY=$EBROOTPYTHON/lib/libpython%%(pyshortver)s.%s " % SHLIB_EXT
+configopts += " -DPYTHON_INCLUDE_DIR=$EBROOTPYTHON/include/python%(pyshortver)s "
+preinstallopts = "mkdir -p %(installdir)s/lib/python2.7/site-packages/ && "
```

Comment:
please use `%(pyshortver)s` rather than hardcoding `2.7` (I'll fix this for the other existing VTK easyconfigs in a separate PR):

```python
preinstallopts = "mkdir -p %(installdir)s/lib/python%(pyshortver)s/site-packages/ && "
```